core/sqf/sql/scripts/sqstart

```diff
@@ -245,6 +245,39 @@ function checkKerberos {
    return 0
 }
 
+function checkUlimit {
+  
+ #check max_pid 
+ userProc=`cat /proc/sys/kernel/pid_max`
+ if [[ $userProc -gt 65535 ]] ; then
+    echoLog ""
+    echoLog "ERROR: detect issue during startup, please make sure your kernel parameter (sysctl) setup correctly"
```

Comment:
Suggested wordsmith: "Error: issue detected during startup. Please make sure your kernel parameter (sysctl) is set up correctly."